test/keys/secp256k1.spec.js

```diff
@@ -210,27 +217,19 @@ describe('crypto functions', () => {
     const sig = await secp256k1Crypto.hashAndSign(privKey, uint8ArrayFromString('hello'))
 
     try {
+      // @ts-ignore
       await secp256k1Crypto.hashAndVerify(privKey, sig, null)
     } catch (err) {
       return // expected
     }
     throw new Error('Expected error to be thrown')
   })
 
-  it('errors when validating a message with an invalid signature', async () => {
```

Comment:
why removing this?

Reply:
Oh I see, so this now returns a boolean `false` instead of throwing an error. Can we flag that breaking change in the commit message?